datalad_deprecated/resources/website/assets/js/main.js

```diff
@@ -515,7 +515,8 @@ function directory(jQuery, md5) {
       // if row is a directory append '/' to name cell
       if (data.type === 'dir' || data.type === 'git' || data.type === 'annex' || data.type === 'uninitialized') {
         var orig = jQuery('td', row).eq(0).html();
-        orig = '<a>' + orig + '/</a>';
+        var traverse = clickHandler(data); 
+        orig = "<a href='"+traverse.next+"'>"+orig+"/</a>";
```

Comment:
```suggestion
        orig = "<a href='" + traverse.next + "'>" + orig + "/</a>";
```
we do not really have style guidelines + some auto-blacking/linting for JS in this project (yet) but it feels that we should get some ;-)

Reply:
BTW, may be while at it -- isn't mighty JS support some XX-century string interpolations? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#expression_interpolation suggests so, so may be this could become
```suggestion
        orig = `<a href='${traverse.next}'>${orig}</a>`;
```
? (I didn't test)

Reply:
Yea, I had considered doing template literals (since they are much easier and cleaner), but wanted to keep it all in es5 in case it broke anything or someone on an old browser suddenly had the links break. Something like Babel would solve this, but I haven't looked much into how create-sibling --ui true actually deploys the site, so I'll hold off for now until I get a better idea on that end. Same with linting, I think I'd need a general package manager like npm for certain integrations on that end, but that seems like a larger project revamp that might be a bit much given the relatively small scale of this site, and changing/needing to see how --ui true deploys the website.